### ISSUE TITLE
v1.2.0 Release Checklist Due 2018-09-11

### ISSUE THREAD

**Issue by dclear** [drud/ddev#1060]
Remaining actions:

* [x]  Review release (product owner)
* [x] Approve release (product owner and release manager)
* [x] Create binaries (any DRUD maintainer)
* [x] Draft release notes (release manager)
* [x] Draft additional announcements for blog, newsletter, etc (not applicable at the moment)
* [x] Create release (release manager)
* [x] Update Roadmap (TPM/release manager)
* [x] Followup issues spun off if we need them.

For additional background information, please see our [Product Release](https://github.com/drud/community/blob/master/development/product_release.md) instructions here.

**Comment by rickmanelius** [comment]
Updated the release date to reflect the cadence of releasing every other sprint.

**Comment by dclear** [comment]
ty @rickmanelius :)

**Comment by rfay** [comment]
Testing with commit 01512b948ddfd51ef89474583715294a5acc7056 on macOS:

- [x] Offer to create docroot in ddev config. (#1048): This worked predictably for me. I went through the process and selected typo3 for the project type and "public" for docroot; it offered to create. It did create. Then I did `ddev ssh` and `cd /var/www/html` and `composer require typo3/minimal` and did an install just fine.
- [x] Empty array in ddev list -j when no projects running. (#1083). This looks fine to me. `ddev list -j` with no projects running gets `    "raw": [],` in the raw section. 
- [x] Specify files dir in config.yaml (#1093): I added a comment that I'd have preferred for the default to show in the config.yaml (don't use omitempty). This seems to work as expected. I did a `ddev import-files` with it changed to sites/default/files_alt and it seems to have worked correctly. I also did a ddev pull on a pantheon project and it put the files in the correct directory.
- [x] apache for ddev (#1007): This seems to work fine, and has been tested by our team and TYPO3 folks. However, as @andrewfrench pointed out, `i=0; while true; do let i=i+1; echo $i; ddev restart || break; done` will break it pretty fast (apache dies or fails to come up). That's the reason our tests failed. 
- [x] Additional config settings from command line (#1092): I tried `ddev config pantheon --http-port=8000 --https-port=8443 --php-version=7.2 --additional-fqdns=something.com --additional-hostnames="one,two,three" --pantheon-environment=dev --xdebug-enabled=true` and it seemed to work correctly; I saw the right things in config.yaml
